release/cli/pkg/assets/assets.go

```diff
@@ -30,10 +34,9 @@ import (
 	"github.com/aws/eks-anywhere/release/cli/pkg/filereader"
 	releasetypes "github.com/aws/eks-anywhere/release/cli/pkg/types"
 	bundleutils "github.com/aws/eks-anywhere/release/cli/pkg/util/bundles"
-	sliceutils "github.com/aws/eks-anywhere/release/cli/pkg/util/slices"
 )
 
-func getAssetsFromConfig(ac *assettypes.AssetConfig, rc *releasetypes.ReleaseConfig, eksDReleaseChannel, eksDReleaseNumber, kubeVersion string) ([]releasetypes.Artifact, error) {
+func getAssetsFromConfig(_ context.Context, ac *assettypes.AssetConfig, rc *releasetypes.ReleaseConfig, eksDReleaseChannel, eksDReleaseNumber, kubeVersion string) ([]releasetypes.Artifact, error) {
```

Comment:
The context param is not being used but it helps with cancellations. In case of `errgroup.WithContext`, the context is cancelled the first time this function returns an error or after all the goroutines have successfully run, which is indicated by a non-nill error from `errgroup.Wait()` method.